PerfettoCds/Pipeline/Tables/PerfettoFrameTable.cs

```diff
@@ -0,0 +1,148 @@
+            new UIHints { Width = 100 });
+
+        private static readonly ColumnConfiguration JankTagColumn = new ColumnConfiguration(
+            new ColumnMetadata(new Guid("{96602fe5-4a2e-4222-986f-baf9aed47ce1}"), "JankTag", "Jank tag"),
+            new UIHints { Width = 100 });
+
+        public static bool IsDataAvailable(IDataExtensionRetrieval tableData)
+        {
+            return tableData.QueryOutput<ProcessedEventData<PerfettoFrameEvent>>(
+                new DataOutputPath(PerfettoPluginConstants.FrameEventCookerPath, nameof(PerfettoFrameEventCooker.FrameEvents))).Any();
+        }
+
+        public static void BuildTable(ITableBuilder tableBuilder, IDataExtensionRetrieval tableData)
+        {
+            // Get data from the cooker
+            var events = tableData.QueryOutput<ProcessedEventData<PerfettoFrameEvent>>(
+                new DataOutputPath(PerfettoPluginConstants.FrameEventCookerPath, nameof(PerfettoFrameEventCooker.FrameEvents)));
+
+            var tableGenerator = tableBuilder.SetRowCount((int)events.Count);
+            var baseProjection = Projection.Index(events);
+
+            var startProjection = baseProjection.Compose(x => x.StartTimestamp);
+            var endProjection = baseProjection.Compose(x => x.EndTimestamp);
+
+            tableGenerator.AddColumn(ProcessNameColumn, baseProjection.Compose(x => x.ProcessName));
+            tableGenerator.AddColumn(ProcessIdColumn, baseProjection.Compose(x => x.Upid));
+            tableGenerator.AddColumn(FrameTypeColumn, baseProjection.Compose(x => x.FrameType));
+            tableGenerator.AddColumn(JankTypeColumn, baseProjection.Compose(x => x.JankType));
+            tableGenerator.AddColumn(AppOnTimeColumn, baseProjection.Compose(x => x.AppOnTime));
+            tableGenerator.AddColumn(DurationColumn, baseProjection.Compose(x => x.Duration));
+            tableGenerator.AddColumn(DisplayTokenColumn, baseProjection.Compose(x => x.DisplayToken));
+            tableGenerator.AddColumn(SurfaceTokenColumn, baseProjection.Compose(x => x.SurfaceToken));
+            tableGenerator.AddColumn(PresentTypeColumn, baseProjection.Compose(x => x.PresentType));
+            tableGenerator.AddColumn(PredictionTypeColumn, baseProjection.Compose(x => x.PredictionType));
+            tableGenerator.AddColumn(GpuCompositionColumn, baseProjection.Compose(x => x.GpuComposition));
+            tableGenerator.AddColumn(JankTagColumn, baseProjection.Compose(x => x.JankTag));
+            tableGenerator.AddColumn(StartTimestampColumn, startProjection);
+            tableGenerator.AddColumn(EndTimestampColumn, endProjection);
+
+
+            var jankFramesConfig = new TableConfiguration("Jank Frames")
+            {
+                Columns = new[]
+                {
+                    ProcessNameColumn,
+                    SurfaceTokenColumn,
+                    FrameTypeColumn,
```

Comment:
Going to leave some comments about changing the default view. Both to make it easier to use & to closer match the Perfetto UI. Changes can be made in the column but will put comments here for readability. FrameType should be sort descending 0.